tests/16-cidr-ingress-policy.sh

```diff
@@ -79,21 +82,59 @@ function test_cidr_except {
             {"matchLabels":{"${ID_SERVICE2}":""}}
         ],
         "fromCIDRSet": [ {
-            "cidr": "${IPV4_PREFIX}",
+            "cidr": "10.0.0.0/8"
+        }
+        ]
+    },{
+        "fromEndpoints": [
+            {"matchLabels":{"${ID_SERVICE2}":""}}
+        ],
+        "fromCIDRSet": [ {
+            "cidr": "10.0.0.0/9",
             "except": [
-                "${IPV4_PREFIX_EXCEPT}"
```

Comment:
I think this removes the last usage of `${IPV4_PREFIX_EXCEPT}`.

Reply:
^ If this is unused, we should either be using it, or just delete it.

Reply:
I did delete it - see lines 70 and 65 which were removed.

Reply:
Ah, perfect. I forget that GitHub doesn't detect stuff like that. Thanks!